src/components/Form.js

```diff
@@ -40,6 +40,9 @@ const propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     draftValues: PropTypes.object,
 
+    /** Should we show the form button */
+    isSubmitButtonVisible: PropTypes.bool,
```

Comment:
I think this needs to be moved below `submitButtonText`, as now it sits under the `/* Onyx Props */` comment but it's not an Onyx prop.

Reply:
this change has been done